v2/fleet-local/control/flight-director@.service

```diff
@@ -7,13 +7,14 @@ Requires=docker.service
 Wants=chronos.service marathon.service
 
 [Service]
-Environment=FD_IMAGE=behance/flight-director:latest
-
 User=core
 Restart=on-failure
 RestartSec=20
 TimeoutStartSec=0
-ExecStartPre=/usr/bin/docker pull ${FD_IMAGE}
+ExecStartPre=/usr/bin/sh -c "docker pull \
+    $(((etcdctl ls images|grep /images/fd > /dev/null) \
```

Comment:
Can't you just do `etcdctl ls /images/gd` instead of grepping?  etcdctl returns a non-zero exit code when it can't find a key @chr0n1x